commands.js

```diff
@@ -2991,7 +2991,25 @@ exports.commands = {
 
 		room.game.undo(user, target);
 	},
-
+	toursexplication: 'toursguide',
+        toursguide: function (target, room, user) {
+  		if (!this.canBroadcast()) return; 
+  		this.sendReplyBox(
+			'<b>Tournament Rules and General Guidelines</b><br>' +
+			'<a href="http://www.smogon.com/forums/threads/tournament-rules-and-general-guidelines.3538684/">List of TD, rules, and more explication.</a><br>' +
+			'<a href="http://www.smogon.com/forums/threads/tournament-rules-and-general-guidelines.3538684/#post-6823208">Definition of Ghosting</a><br>' +
+			'<a href="http://www.smogon.com/forums/threads/tournament-rules-and-general-guidelines.3538684/#post-6703482">Guide to Scheduling</a><br>' +
+			'<hr>' +
+			'<b>List of Official Smogon Tournaments</b><br>' +
+			'<img src="http://www.smogon.com/media/forums/images/badges/tour.png" width="16" height="16"><a href="http://www.smogon.com/forums/forums/smogon-tour.49/">Smogon Tour</a><br>' +
+			'<img src="http://www.smogon.com/media/forums/images/badges/spl.png" width="16" height="16"><a href="http://www.smogon.com/forums/forums/smogon-premier-league.130/">Smogon Premier League</a><br>' +
+			'<img src="http://www.smogon.com/media/forums/images/badges/worldcup.png" width="16" height="16"><a href="http://www.smogon.com/forums/forums/world-cup-of-pokemon.234/">Worl Cup of Pokemon</a><br>' +
+			'<img src="http://www.smogon.com/media/forums/images/badges/grandslam.png" width="16" height="16"><a href="http://www.smogon.com/forums/forums/smogon-grand-slam.208/">Smogon Grand Slam</a><br>' +
+			'<img src="http://imgur.com/nIdW9QW.png" width="16" height="16"><a herf="http://www.smogon.com/forums/forums/the-smogon-classic.334/">The Smogon Classic</a><br>' +
+			'<img src="http://www.smogon.com/media/forums/images/badges/olt.png" width="16" height="16"><a href="http://www.smogon.com/forums/threads/smogons-official-ladder-tournament-iii-cycle-4-signups-read-post-178.3580529/#178">Smogons Official Ladder Tournament (cycle #4)</a><br>' +
+			'<img src="http://www.smogon.com/media/community/awards/ost.png" width="16" height="16"><a href="http://www.smogon.com/forums/threads/smogon-tournament-12-signups.3562131/">Smogon Official Smogon Tournament</a>'
+		); 
+	},
```

Comment:
There's a trailing space in line 3011 ``);``.

Line 3012: At the beginning you use a space & then tab, remove that space and just use tab.